app/decorators/controllers/solidus_marketplace/spree/admin/reports_controller_decorator.rb

```diff
@@ -34,21 +34,67 @@ def earnings_csv
           end
         end
 
+        def supplier_total_sales
+          params[:q] = search_params
+
+          @search = ::Spree::Order.complete.not_canceled.ransack(params[:q])
```

Comment:
is there any default params? like only last 30 days, I can see some performance degradation over time with more and more data being computed

Reply:
Makes sense to add a range, will update that